charts/datadog/templates/container-process-agent.yaml

```diff
@@ -22,9 +22,9 @@
     - name: DD_SYSTEM_PROBE_ENABLED
       value: {{ .Values.datadog.systemProbe.enabled | quote }}
     {{- end }}
-    {{- if .Values.datadog.orchestratorExplorer.enabled }}
     - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
-      value: "true"
+      value: {{ .Values.datadog.orchestratorExplorer.enabled | quote }}
+    {{- if .Values.datadog.orchestratorExplorer.enabled }}
```

Comment:
just tested this with `template`. It still works. If the explorer is set to false, it will not reference the `datadog-cluster-id`